automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/metadata/DataSpec.scala

```diff
@@ -35,10 +35,10 @@ class DataSpec extends FreeSpec with ParallelTestExecution with WebBrowserSpec w
 
           withWebDriver { implicit driver =>
             withSignIn(owner) { _ =>
+              api.workspaces.waitForBucketReadAccess(billingProject, workspaceName)
```

Comment:
If I was doing this, I would move `waitForBucketReadAccess` to line # 35, before `withWebDriver`. I'm skeptical this change is going to make any difference. But it worth a shot.

Reply:
the pattern of `waitForBucketReadAccess` inside of `withWebDriver` is present in about 6 other tests, so it felt safe - happy to move it, though. The majority of tests beyond those 6 have it outside `withWebDriver`.